pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -950,7 +952,9 @@ func appNetworkDoActivateUnderlayNetwork(
 	if err != nil {
 		addError(ctx, status, "createACL", err)
 	}
-	ulStatus.ACLRules = ruleList
+	appID := status.UUIDandVersion.UUID
+	getNetworkACLRules(ctx, appID, ulStatus.Name)
```

Comment:
This may not be required here if setNetworkACLRules handles the new app UUID case as per my comment above.

Reply:
sure.